client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -135,13 +137,27 @@ export class CreditCardPaymentBox extends React.Component {
 				overSome( isWpComBusinessPlan, isWpComEcommercePlan )( product_slug )
 			),
 			showPaymentChatButton = presaleChatAvailable && hasBusinessPlanInCart,
-			paymentButtonClasses = 'payment-box__payment-buttons';
+			paymentButtonClasses = 'payment-box__payment-buttons',
+			moneyBackGuarantee =
+				! hasOnlyDomainProducts( cart ) && 'variantShowGuarantee' === abtest( 'checkoutGuarantee' );
 
 		return (
 			<div className={ paymentButtonClasses }>
 				<PayButton cart={ cart } transactionStep={ transactionStep } />
 
 				<div className="checkout__secure-payment">
+					{ moneyBackGuarantee && (
+						<div className="checkout__secure-payment-content">
```

Comment:
I see an alignment issue on mobile:

<img width="292" alt="Screenshot 2019-07-17 at 12 59 50 PM" src="https://user-images.githubusercontent.com/1269602/61355863-d5f7f700-a892-11e9-9c75-b577cbc070cd.png">

I've [pushed](https://github.com/Automattic/wp-calypso/pull/34677/commits/546a1c8562edb08e3cd46053793cffab38e0c6af) a small refactor to fix this, along with removing the `<br />` element entirely.

Reply:
Awesome, thanks a lot!